front-end/src/components/IntentLister.jsx

```diff
@@ -144,25 +144,27 @@ class IntentLister extends Component {
                     </div>
                     }
                 </div>
-                <div className="fixed left-4 bottom-8">
+                <div className="fixed left-6 bottom-8">
                     <BaseButton
                         click={this.decrementIndex}
                         isDisabled={this.checkIfMinIndex()}
                         icon={{
                             name: this.state.keysHeld["Control"] ? "skip-left" : "arrow-left",
                             size: "40"
                         }}
+                        tooltip="Click this button or press the ← key while holding control to skip to the beginning"
```

Comment:
Can we call control "CTRL" or some other thing to indicate that it is a key name? idk might be easier on first glance (same ideas as the left arrow icon vs saying "Left Arrow")

Reply:
Great idea